doc/source/cluster/configure-manage-dashboard.md

```diff
@@ -0,0 +1,306 @@
+(observability-configure-manage-dashboard)=
+
+# Configuring and Managing Ray Dashboard
+{ref}`Ray Dashboard<observability-getting-started>` is one of the most important tools to monitor and debug Ray applications and clusters. This page describes how to configure Ray Dashboard on your clusters.
+
+Dashboard configurations may differ depending on how you launch Ray clusters (e.g., local Ray cluster v.s. KubeRay). Integrations with Prometheus and Grafana are optional for better dashboard experience.
+
+:::{note}
+At the moment, Ray Dashboard is only intended for interactive development and debugging because dashboard UI and the underlying data are not accessible any more after the clusters are terminated. For production monitoring and debugging, users should rely on [persisted logs](../cluster/kubernetes/user-guides/logging.md), [persisted metrics](./metrics.md), [persisted Ray states](../ray-observability/user-guides/cli-sdk.rst), and other observability tools.
+:::
+
+## Changing Ray Dashboard Port
+Ray Dashobard runs on port `8265` of the head node. Follow the instructions below to customize the port if needed.
+
+::::{tab-set}
+
+:::{tab-item} Single-node local cluster
+**Start the cluster explicitly with CLI** <br/>
+To customize the port on which the dashboard runs, pass
+the ``--dashboard-port`` argument with ``ray start`` in the command line.
+
+**Start the cluster implicitly with `ray.init`** <br/>
+Pass the keyword argument ``dashboard_port`` in your call to ``ray.init()``.
+:::
+
+:::{tab-item} VM Cluster Launcher
+To be added
```

Comment:
@rkooo567 can you write a few sentences for this? The 2.4 doc about it is wrong.